__tests__/classes/CanvasRenderingContext2D.fillStyle.js

```diff
@@ -11,7 +11,7 @@ beforeEach(() => {
 describe('fillStyle', () => {
   it('should parse a css color string \'blue\'', () => {
     ctx.fillStyle = 'blue';
-    expect(ctx.fillStyle).toBe('#00f');
+    expect(ctx.fillStyle).toBe('blue');
```

Comment:
Please follow the specification. The browser returns `#00f`, and that's why this value is what it is.